.github/workflows/automation-benchmark-tests.yml

```
@@ -75,6 +75,8 @@ jobs:
           INTERNAL_DOCKER_REPO: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com
           TEST_TYPE: ${{ github.event.inputs.testType }}
           TEST_TEST_TYPE: ${{ github.event.inputs.testType }}
+          RR_MEM: 4Gi
+          TEST_LOG_LEVEL: debug
```

```suggestion
          RR_MEM: 4Gi
```
Debug mode can be disabled  now.